packages/compose/src/hooks/use-media-query/index.js

```diff
@@ -33,5 +33,5 @@ export default function useMediaQuery( query ) {
 		};
 	}, [ query ] );
 
-	return query && match;
+	return !! query && match;
```

Comment:
This seems to be breaking some existing tests.

Reply:
Hmm, you're right. I wonder if we should update the test or keep the existing behavior and use a wider return type 🤔

Reply:
I've gone ahead and updated the test.